pom.xml

```diff
@@ -36,9 +36,26 @@
         <dependency>
             <groupId>com.h2database</groupId>
             <artifactId>h2</artifactId>
-            <version>1.4.196</version>
+            <version>1.4.199</version>
         </dependency>
-<!--        <dependency>-->
+        <dependency>
```

Comment:
Do we need Junit? 
If not is it bad if we have unused libraries?

Reply:
I guess no. If we need, we'll put it back.